gnomad/utils/transcript_annotation.py

```diff
@@ -53,48 +52,122 @@ def summarize_transcript_expression(
     mt = mt.group_cols_by(tissue=tissue_expr).aggregate(
         tx=summary_agg_func(transcript_expression_expr)
     )
+    ht = mt.rename({"tx": ""}).make_table().key_by("transcript_id", "gene_id")
+
+    # Annotate with the proportion of expression of transcript to gene per tissue.
```

Comment:
Since this now includes the proportion, and the structure of the output table is different, I think this statement in the docstring needs to change, and maybe have an example of the output schema

```
The returned Table has a row annotation for each tissue containing the summarized
tissue expression value.
```